src/main/res/values/strings.xml

```diff
@@ -350,9 +350,9 @@
     <string name="opal_action_new_journey">New journey</string>
     <string name="opal_action_transfer_same_mode">Transfer (from same mode)</string>
     <string name="opal_action_transfer_diff_mode">Transfer (from different mode)</string>
-    <string name="opal_action_manly_new_journey">New journey (Manly Ferry)</string>
-    <string name="opal_action_manly_transfer_same_mode">Transfer (from other ferry to Manly Ferry)</string>
-    <string name="opal_action_manly_transfer_diff_mode">Transfer (from other mode to Manly Ferry)</string>
+    <string name="opal_action_manly_new_journey">New journey (Manly ferry)</string>
```

Comment:
The usage of "Manly Ferry" was correct in this context, as it is referring to a specific ferry line (F1), which is also distinct from "Manly Fast Ferry" (the name of a different line, and a private operator).

See also:

- https://transportnsw.info/documents/timetables/09-F1-Manly-20171126.pdf
- https://transportnsw.info/news/2017/contactless-payments-on-sydney-ferries-f1-manly-ferry